src/lib/sm.c

```diff
@@ -49,6 +60,7 @@ void sm_init(struct sm *m,
 	m->is_locked = is_locked;
 	m->id = ++id;
 	m->pid = getpid();
+	m->rc = 0;
```

Comment:
Note: using initializers can help not to think about setting to zero unset attributes:
```
*m = {.conf = conf, .state=state ..};
```